src/app/zap-templates/partials/im_command_handler_cluster_commands.zapt

```diff
@@ -0,0 +1,38 @@
+{{#if (zcl_command_arguments_count this.id)}}
+CHIP_ERROR TLVError = CHIP_NO_ERROR;
+{{#zcl_command_arguments}}
+{{#if (isString type)}}
+const uint8_t * {{asSymbol label}};
+{{else}}
+{{asUnderlyingZclType type}} {{asSymbol label}};
+{{/if}}
+{{/zcl_command_arguments}}
+
+while ((TLVError = dataTlv.Next()) == CHIP_NO_ERROR)
+{
+  switch (TLV::TagNumFromTag(dataTlv.GetTag()))
+  {
+{{#zcl_command_arguments}}
+    case {{index}}:
+{{#if (isString type)}}
+      TLVError = dataTlv.GetDataPtr({{asSymbol label}});
+{{else if isArray}}
+      // Just for compatibility, we will add array type support in IM later.
+      TLVError = dataTlv.GetDataPtr(const_cast<const uint8_t *&>({{asSymbol label}}));
+{{else}}
+      TLVError = dataTlv.Get({{asSymbol label}});
+{{/if}}
+      break;
+{{/zcl_command_arguments}}
+    default:
+      // Unsupported tag, ignore it.
+      // TODO: Can be a log for INFO level
```

Comment:
please add logs here and below.

Reply:
Yes, since this is just a single line of code.